src/theme/variables.css

```diff
@@ -260,6 +260,47 @@ ion-thumbnail > img {
   margin-top: 25px;
 }
 
+.scanner {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(343px, 1fr));
+  gap: 8px;
```

Comment:
Instead of pixel values please use spacer variables where we have provided spaces.

Reply:
Used spacer variables.